src/exchange/offerMake/OfferMakeForm.stories.tsx

```diff
@@ -305,25 +308,26 @@ stories.add('Validations on form', () => {
     </Panel>
 
     </div>
+    </BrowserRouter>
   );
 });
 
 stories.add('Three validations error on one element', () => {
   const messages: Message[] = [
     {
-      kind: MessageKind.noAllowance,
+      kind: MessageKind.insufficientAmount,
```

Comment:
Probably I"m missing the idea but why did you change that?

Reply:
I could alternatively wrap the form with a Router, but decided only to wrap the story which displays all possible errors, and skip other stories.